apps/taxes/src/modules/avatax/maps/avatax-calculate-taxes-map.ts

```diff
@@ -62,49 +62,88 @@ const mapPayload = (props: AvataxCalculateTaxesMapPayloadArgs): CreateTransactio
         shipTo: avataxAddressFactory.fromSaleorAddress(taxBase.address!),
       },
       currencyCode: taxBase.currency,
-      lines: mapLines(taxBase, config),
+      lines: mapPayloadLines(taxBase, config),
       date: new Date(),
     },
   };
 };
 
-const mapResponse = (transaction: TransactionModel): CalculateTaxesResponse => {
+export function mapResponseShippingLine(
+  transaction: TransactionModel
+): Pick<
+  CalculateTaxesResponse,
+  "shipping_price_gross_amount" | "shipping_price_net_amount" | "shipping_tax_rate"
+> {
   const shippingLine = transaction.lines?.find((line) => line.itemCode === SHIPPING_ITEM_CODE);
 
-  const productLines = transaction.lines?.filter((line) => line.itemCode !== SHIPPING_ITEM_CODE);
+  if (!shippingLine?.isItemTaxable) {
+    return {
+      shipping_price_gross_amount: shippingLine?.lineAmount ?? 0,
+      shipping_price_net_amount: shippingLine?.lineAmount ?? 0,
```

Comment:
@peelar please explain it. If `lineAmoount` doesnt exist, it will return 0. But its not tax amount, but the product amount?
If item is not taxable, gross === net, right? But not zero?